.vscode/settings.json

```
@@ -58,5 +58,9 @@
     "json",
     "jsonc",
     "yaml"
-  ]
+  ],
+  "githubPullRequests.ignoredPullRequestBranches": [
+    "main"
+  ],
+  "cSpell.language": "en,el"
```

```suggestion
  ]
```

Please avoid committing this workspace setting file since this is shared by other developers globally.

I think you're changing "Workspace setting" in VS Code. You can use "User setting" instead for your local setting to avoid changing this file by the editor.

ref. https://code.visualstudio.com/docs/getstarted/settings